.github/README.md

```diff
@@ -39,6 +40,81 @@ cd  # in your home directory
 git clone https://github.com/torvalds/linux.git
 ```
 
+If you are compiling the kernel for the first time, install some of the packages required:
+
+On a Debian/Ubuntu distribution:
+
+
+```
+sudo apt install -y \
+  qtbase5-dev \
+  pkg-config \
+  bison \
+  flex \
+  libelf-dev \
+  libssl-dev \
+  libcap-dev \
+  llvm
```

Comment:
You recommend installing llvm but then later get the user to compile their own llvm. I don't think this one is necessary.

Reply:
Removed.